numpy/core/multiarray.py

```diff
@@ -0,0 +1,27 @@
+from . import _multiarray_umath
```

Comment:
I think this file could do with either a docstring or a comment, explaining why it exists and is just a bunch of imports